cluster/ClusterDatabase.java

```diff
@@ -55,10 +57,10 @@ private ClusterDatabase(String database, ClusterDatabaseManager clusterDatabaseM
         }
     }
 
-    static ClusterDatabase of(ClusterDatabaseProto.ClusterDatabase protoDB, ClusterDatabaseManager clusterDatabaseMgr) {
+    static ClusterDatabase of(ClusterDatabaseProto.ClusterDatabase protoDB, ClusterClient client, ClusterDatabaseManager clusterDatabaseMgr) {
```

Comment:
see above

Reply:
Fixed in dc11f37cc5faade1de8a9f377923eb87269439f3.